Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM fedora
+# Normally it's illadvised to use a FQDN, but this image is auto-built
+# on quay.io which has no setting to specify a non-docker default registry
+# or build-args.
+ARG REGISTRY=registry.fedoraproject.org/
+FROM ${REGISTRY}fedora:latest
```

Comment:
Absolutely non-blocking: Do we expect anyone to override the variable? Just hard-coding `registry.fedoraproject.org/fedora:latest` might be good enough.

Reply:
TBH I didn't/don't know who/what uses this Dockerfile, so I made the more conservative choice.  I have no problem with "good enough" if that's what it is :grin: